threshold_tools_application_examples.ipynb

```diff
@@ -1168,7 +1163,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.7"
+   "version": "3.8.3"
```

Comment:
@bkg Any thoughts on what's going on here with the python version change? I just deleted some cells, not sure why this is occurring

Reply:
It records the Python version used which is 3.9.7 on the pangeo notebook image, but it sounds like you probably have 3.8.3 locally.

Reply:
bleh. I think I'll try making another PR. Thanks for your help!